dev-site.yml

```diff
@@ -8,24 +8,41 @@ site:
 
 content:
   sources:
-    - url: .
-      branches: HEAD
+    - url: https://github.com/redhat-scholars/tekton-tutorial.git
+      # Stay on master to pull latest changes
+      branches: master
```

Comment:
You still pulling master. PRs to this repo that reference another repo should reference a specific tag. Otherwise you have no idea what you're reviewing.

Reply:
The `.ts` file is used to do live build and test of the site `openshift-devops-admins` when doing locally, its  more a convenience, it does not bother anything in the site gen. Not sure how a build of a local site can be performed by pulling task file from other repo. 

Also note  - we are importing content only and not how we build ...

Reply:
Since this live reload way is bit advanced , I did not put in the course build doc and not force on others.

Reply:
@thoraxe  removed all local build files and tagged the tekton content